ui/plugins/controller.js

```diff
@@ -150,6 +150,14 @@ treeherder.controller('PluginCtrl', [
                     // exclude the json log URLs
                     $scope.job_log_urls = _.reject(results[3], {name: 'mozlog_json'});
 
+                    // Provide a parse status as a scope variable for logviewer shortcut
+                    if (!$scope.job_log_urls.length) {
+                        $scope.logParseStatus = 'unavailable';
+                    } else if ($scope.job_log_urls[0].parse_status) {
+                        $scope.logParseStatus = $scope.job_log_urls[0].parse_status;
+                    }
```

Comment:
This new `logParseStatus` I guess is not really needed; it was more to prettify' the notification block below, so the tested variable was the same for each case. I think I can yank it if needed. I'm also still a bit unclear whether `job_log_urls[0]` is sufficient (can there be multiple logs for a job?), but it seems to be ok in testing.